packages/schematics/angular/library/files/__projectRoot__/tsconfig.lib.json

```diff
@@ -0,0 +1,29 @@
+{
+  "extends": "<%= projectRoot.split('/').map(x => '..').join('/') %>/tsconfig.json",
+  "compilerOptions": {
+    "outDir": "",
```

Comment:
We should add an outdir for when ngc is used directly (we were debugging the error message as well and ran into it).

On another node, we could not reproduce the error locally... were you able to?

Reply:
Good point, re the `outDir`,  actually I got mislead by the `hello world app` failing unit test. I am still waiting for the tests to run. Is there any short cut to run a specific set of N tests?

Reply:
Update I did manage to replicate it 
<img width="1444" alt="screen shot 2018-04-25 at 20 59 33" src="https://user-images.githubusercontent.com/17563226/39266894-9793fb8a-48cb-11e8-921e-413722af5a53.png">

I will have to look at this tomorrow.